docs/modules/qualitybandwidth_connectivity.html

```diff
@@ -289,7 +289,11 @@ <h4>Configure a video downlink policy</h4>
 					</a>
 					<p>By default the SDK uses the <a href="https://github.com/aws/amazon-chime-sdk-js/blob/master/src/videodownlinkbandwidthpolicy/AllHighestVideoBandwidthPolicy.ts">AllHighestVideoBandwidthPolicy</a> which subscribes to the highest quality video of all participants. This can be customized by setting the <a href="https://github.com/aws/amazon-chime-sdk-js/blob/master/src/meetingsession/MeetingSessionConfiguration.ts#L74">VideoDownlinkBandwidthPolicy</a> in MeetingSessionConfiguration class.</p>
 					<p><em>Browser clients currently only send one stream resolution. You would only need to use this function if you were also using the Amazon Chime SDK for iOS or the Amazon Chime SDK for Android.</em></p>
-					<p><a href="https://github.com/aws/amazon-chime-sdk-js/issues/new?assignees=&amp;labels=documentation&amp;template=documentation-request.md&amp;title=Quality%20Bandwidth_Connectivity%20feedback">Give feedback on this guide</a></p>
+					<a href="#use-video-simulcast" id="use-video-simulcast" style="color: inherit; text-decoration: none;">
```

Comment:
I also thought we can use the downlink policy WITHOUT enabling simulcast - did we test that configuration and can we offer that here?

Reply:
The feature flag bundles simulcast uplink policy and downlink policy together. 
Developers could indeed use downlink policy by providing it into configuration directly, however the downlink policy only offer limited help without simulcast, all it can do is to drop stream without simulcast, which I think it's redundant to "pausing video stream" solution and can be confusing.

Reply:
I agree, though i would think its still valuable